engine/pikachu/pikachu_status.asm

```diff
@@ -20,10 +20,11 @@ IsStarterPikachuInOurParty::
 	ld a, [wPlayerID+1]
 	cp [hl]
 	jr nz, .curMonNotPlayerPikachu
+
 	push de
 	push bc
 	ld hl, wPlayerName
-	ld d, $6 ; possible player length - 1
+	ld d, $6 ; leftover hardcoded Japanese NAME_LENGTH
```

Comment:
Use `NAME_LENGTH_JP` like pokegold/pokecrystal. (Actually those call it `NAME_LENGTH_JAPANESE`, but that should be renamed, we use `_JP` in various other places.)